ddmrp/models/stock_buffer.py

```diff
@@ -978,10 +978,14 @@ def _past_demand_estimate_domain(self, date_from, date_to, locations):
 
     def _past_moves_domain(self, date_from, date_to, locations):
         self.ensure_one()
+        # Exclude inventory loss moves
+        inventory_locations = self.env["stock.location"].search(
+            [("usage", "=", "inventory")]
+        )
         return [
             ("state", "=", "done"),
             ("location_id", "in", locations.ids),
-            ("location_dest_id", "not in", locations.ids),
+            ("location_dest_id", "not in", (locations | inventory_locations).ids),
```

Comment:
can't we save a search by using `("location_dest_id.usage", "in", (...allowed_values...))`?

Reply:
Good point, but using a dotted syntax in a search domain doesn't generate another query too?

Reply:
After checking the execution query plan of both requests with @guewen , the one with `location_dest_id.usage` could be better at the end with lots of moves, it needs confirmation but I prefer to change it this way for now as it also makes the code clearer about the intent.